tests/rptest/tests/describe_topics_test.py

```diff
@@ -313,6 +313,13 @@ def test_describe_topics_with_documentation_and_types(self):
                 value="dlq_table",
                 doc_string=
                 "Action to take when an invalid record is encountered."),
+            "redpanda.iceberg.target.lag.ms":
+            ConfigProperty(
+                config_type="LONG",
+                value="300000",
```

Comment:
this isn't 1 minute 🤔

Reply:
ah, i meant to remove this. the config property cases here for iceberg & schema validation are actually dead code, since describe topics only returns the configured values if the corresponding feature is switched on!